packages/orchestration/src/orchestration.js

```diff
@@ -156,9 +137,10 @@ const prepareChainAccount = zone =>
           // XXX parseAddress currently throws, should it return '' instead?
           this.state.accountAddress = parseAddress(remoteAddr);
         },
-        async onClose(_connection, reason) {
+        async onClose(_connection, reason, _connectionHandler) {
```

Comment:
why add the unused param? I went the opposite direction in https://github.com/Agoric/agoric-sdk/pull/9191/commits/44107b933fea3c2e231f54e1dec29d31d39f8571